tests/libfixmath/main.c

```diff
@@ -190,6 +190,6 @@ int main(void)
     puts("Binary.");
     binary_ops();
 
-    puts("Done.");
+    puts("SUCCESS");
```

Comment:
it looks like that more tests print out `[SUCCESS]` instead of `SUCCESS`. To be in line with the styling, I would propose to change this here to `[SUCCESS]` and probably update all others to `[SUCCESS]` in a follow up PR.

Reply:
I think it is more important that tests return proper status codes (which is more or less guaranteed when using testrunner) then a unified output format (I could even imagine deactivating the output for `make test` all together). I guess here because I'm on mobile right now, but I would estimate there are as many `[DONE]` as there are `[SUCCESS]` and putting more (IMHO nit-picky) rules on the - for most developers - already annoying part of testing makes it even less fun.